compose/cli/main.py

```diff
@@ -170,6 +170,11 @@ def perform_command(self, options, handler, command_options):
         with friendly_error_message():
             handler(project, command_options)
 
+    def exit_if(self, condition, message, exit_code):
+        if condition:
+            log.error(message)
+            sys.exit(exit_code)
```

Comment:
I also wonder if it would be more correct  to raise an exception here, to be handled by `main()` above, which already does a bunch of this kind of thing.  That way we'd only have `sys.exit()` in a single function.